bin/initdb_local.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
If this is just a debug script, I think it should be named as such so the next person coming in knows. I would also add a comment or something for what variables that `.env` file should have in it.